js/memory.js

```diff
@@ -1,19 +1,35 @@
 class MemoryGame {
   constructor(cards) {
     this.cards = cards;
+    this.pickedCards = [];
+    this.pairsClicked = 0;
+    this.pairsGuessed = 0;
     // add the rest of the class properties here
   }
 
   shuffleCards() {
-    // ... write your code here
```

Comment:
I see you created the pickedCards array but I don't see any use for it

Maybe you could create something like a selectCard method to push the card into the array and the use the array to check wether you have a guess or not